quantum/pointing_device/pointing_device_drivers.c

```diff
@@ -391,7 +391,7 @@ const pointing_device_driver_t pointing_device_driver = {
 };
 #elif defined(POINTING_DEVICE_DRIVER_pimoroni_trackball)
 
-mouse_xy_report_t pimoroni_trackball_adapt_values(clamp_range_t* offset) {
+mouse_xy_report_t pimoroni_trackball_adapt_values(xy_clamp_range_t* offset) {
```

Comment:
Has been included in [a separate PR](https://github.com/qmk/qmk_firmware/pull/24422).